build/buildcheck.sh

```diff
@@ -28,16 +28,16 @@ fi
 # autoconf 2.68 or newer
 ac_version=`$PHP_AUTOCONF --version 2>/dev/null|head -n 1|sed -e 's/^[^0-9]*//' -e 's/[a-z]* *$//'`
 if test -z "$ac_version"; then
-echo "buildconf: autoconf not found."
-echo "           You need autoconf version 2.68 or newer installed"
-echo "           to build PHP from Git."
+echo "buildconf: autoconf not found." >&2
+echo "           You need autoconf version 2.68 or newer installed" 1>&2
+echo "           to build PHP from Git." >&2
 exit 1
 fi
 IFS=.; set $ac_version; IFS=' '
 if test "$1" = "2" -a "$2" -lt "68" || test "$1" -lt "2"; then
-echo "buildconf: autoconf version $ac_version found."
-echo "           You need autoconf version 2.68 or newer installed"
-echo "           to build PHP from Git."
+echo "buildconf: autoconf version $ac_version found." >&2
+echo "           You need autoconf version 2.68 or newer installed" 1>&2
```

Comment:
`>&2` and `1>&2` seems to be mixed here as well.

Reply:
```suggestion
echo "           You need autoconf version 2.68 or newer installed" >&2
```

Reply:
I fixed.